src/main/java/uk/ac/cam/cl/dtg/segue/api/ContactFacade.java

```diff
@@ -128,13 +129,18 @@ public Response contactUs(final Map<String, String> form, @Context final HttpSer
                             .put("contactUserRole", currentUserRole)
                             .put("contactEmail", form.get("emailAddress"))
                             .put("contactSubject", form.get("subject"))
-                            .put("contactMessage", message)
+                            .put("contactMessage", StringEscapeUtils.escapeHtml4(message))
```

Comment:
This feels like it double-escapes things, since we already call `StringEscapeUtils.escapeHtml4` in `EmailManager::sanitizeEmailParameters` on this property :/